Kraken jokes' Pteron56 files

Kraken-Jokes opened the pull request:
Included in this patch are the following files:

**/case:**
Pteron56_MXholes 7mm padding.dxf             _//regular MX style hole plate_
Pteron56_sharpMXholes 7mm padding.dxf    _//MX holes with sharp corners. Less expensive plate design from some shops_
Pteron56_base 7mm padding.dxf                   _//base plate_
Pteron56_stack plate 7mm padding.dxf         _//perimeter plates, 5mm width_

**/images:**
Pteron56_logo.png      _//Logo I used for my pteron build_
Pteron56_logo.svg       _//Logo I used for my pteron build_

**/builds:**
Pteron56 face.jpg         _//My first Pteron build_
Pteron56_ wiring.jpg    _//My first Pteron build from the back. This file isn't necessarily needed, your wiring was much nicer_
                                     _//looking._

Kraken-Jokes committed:
Pteron56 built by Kraken-Jokes

Laser cut 3mm acrylic with custom logo/ art laser etching.
Gateron Red switches, pro micro clone, 8mm spacers

Kraken-Jokes committed:
Adding no-kerf .dxf files

Adding no-kerf files, mx style opening holes with 7mm padding between the switch hole and edge of the keyboard. Use this style only for laser cut materials that are 1.5mm, or .06" thickness.
Base is the base. Stack plates are just perimeter plates for adding thickness to the keyboard. The stack plates are not needed unless you want to add them.
Fastener holes are 2mm diameter.

Kraken-Jokes committed:
Logo I designed for my Pteron56

The font of the logo in my picture in /builds is Another America by The Branded Quotes.
Since that is a paid font I did not use that in this .svg. I used the font Edo by Vic Fieger available here 
https://www.fontsquirrel.com/fonts/Edo?q%5Bterm%5D=edo&q%5Bsearch_check%5D=Y

Also added is the .png version of the logo.

Kraken-Jokes committed:
MX switch opening, no round corners

I made this as a little experiment. When comparing the price of plates from sendcutsend.com, this sharp edged MX style cutout consistently was quoted at on average 15-20% less expensive than the traditional MX style opening when comparing against multiple materials, although the same comparison on oshcut.com showed a slightly higher price for the sharp corner version than the normal MX. This leads me to believe that different laser systems prefer lines and sharp corners over curves and vice versa.

FSund commented:
Hi! Thanks for the PR! Don't think I've ever had anyone contribute to any of my repos before 😅

I don't know if the logos you've designed are that relevant for me, so perhaps you could remove those?

It would also be great if you could add a section to `builds/builds.md` with your images in it.

Kraken-Jokes committed:
Delete Pteron56_logo.svg

Tangental to /main, so unincluded

Kraken-Jokes committed:
Delete Pteron56_logo.png

Tangental to /main to unincluded

Kraken-Jokes committed:
Updated builds.md

Added Kraken-Jokes section in builds.md

Kraken-Jokes committed:
Updated builds.md

Fixed broken link

Kraken-Jokes commented:
Well that's awesome cause this is my first contribution to a repo! (I'm working on split plates for the pteron, they're not ready for review yet though)
I'm new to github, so it took me a bit to get everything squared away. I added my section to the builds.md file and removed those logo images.
I gotta say again, thanks for this design. It's visually amazing and types very comfortably

FSund commented:
Thanks for the contribution!
